src/Apps/Sell/Components/BottomFormNavigation.tsx

```diff
@@ -52,14 +52,14 @@ const BottomFormNextButton = () => {
   const { isValid, isSubmitting, submitForm } = useFormikContext()
   const {
     actions,
-    state: { isLastStep },
+    state: { isSubmitStep },
   } = useSellFlowContext()
 
   const onNext = async () => {
     try {
       await submitForm()
 
-      isLastStep ? actions.finishFlow() : actions.goToNextStep()
+      isSubmitStep ? actions.finishFlow() : actions.goToNextStep()
```

Comment:
I've moved the "Thank You" page inside the steps (at the end) to render the progress bar correctly. This means that we need to define the step that submits the submission differently because it's not the last step anymore.